MethodFunction/MethodFunction/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MethodFunction
+{
+    class Program
+    {
+        static void Main()
+        {
+            var x = Math.Pow(Math.E, F(2, 3)) + (Math.Pow(Math.E, F(3, 5))) * (Math.Pow(Math.E, F(5, 7)));
```

Comment:
Видите, что выражение слишком сложное и страдает повторяемостью кода.
Возведение _e_ в степень лучше было вынести в метод F. 
```suggestion
            var x = F(2, 3) + F(3, 5) * F(5, 7);
```